src/site/content/en/patterns/web-vitals-patterns/notices-pattern-set/notice-sticky-footer/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: pattern
+title: Sticky footer
+description: Display a notice as a sticky footer
+date: 2021-08-10
+height: 300
+---
+
+Sticky footers are a popular way of displaying notices. Notices placed in the footer are less likely to obscure page elements like navigation bars, branding, and banner ads. In addition, sticky footers do not cause layout shifts when they are inserted in to the page.
```

Comment:
s/inserted in to/inserted into/

Also I think line wrapping needs to be applied to this.